### ISSUE TITLE
Use abandonwares noble fork

### ISSUE THREAD

**Issue by zilluss** [henry-spanka/nubli#2]
Apparently, noble is not maintained anymore (See https://github.com/noble/node-bluetooth-hci-socket/issues/84). I noticed this when I tried to install nubli-homebridge on my Raspberry 3 B, which failed to build on Node 10. I would suggest to switch to @abandonware/noble, which appears to be a well maintained fork.

**Comment by henry-spanka** [comment]
Hi,
looks reasonable. Can you install nubli if you replace the package vendor in package.json?

**Comment by zilluss** [comment]
Yes, I just tried it. I also tried building the unmodified checkout to confirm the build error. noble 1.9.1 fails with:
binding.target.mk:99: recipe for target 'Release/obj.target/binding/src/BluetoothHciSocket.o' failed

Changing to @abandonware/noble builds just fine. I had to bump the version to 1.9.2-2 though.

**Comment by zilluss** [comment]
While it does compile, it doesn't work quite right (yet), presumably related to this issue:
https://github.com/abandonware/noble/issues/15
I was able to pair with my lock (although only with bridge=false). But any operation that requires pairing won't work. I will post an update once I got everything running using Node 10 and Linux/Raspberry.

**Comment by henry-spanka** [comment]
Okay. I am currently getting my hands on the new ODROID N2. I will probably switch to Node 10/11 too and will try to use abandonware's fork instead.

**Comment by henry-spanka** [comment]
I couldn't get it working either. I think we must wait until the issue is resolved in the upstream repo. Alternatively you can run node 8 in a docker container, that's what I did now.

**Comment by zilluss** [comment]
I tested nubli on Node 12 with @abandonware/noble and it works fine (using an USB BT dongle). May I create a PR that uses the abandonware package instead?

**Comment by henry-spanka** [comment]
Please do. Unfortunately I don't have a Nuki smart lock anymore so I can not test it.

**Comment by muratcorlu** [comment]
@zilluss I couldn't see any PR. Are you still able to use it? Does it also work with Node 14?

**Comment by zilluss** [comment]
I didn't use this plugin in a while, I will test it and get back at you. The problem at the time was that I had to create a PR at the noble fork with additions to the typescript bindings, which are merged by now. So hopfefully using the abandonware fork should be as easy as replacing a line in the package.json.
